Bottom: 556bbaf0d43992e2337e2d8ed37d6c09c54778ca
Top:    17c3343cb4d153928593ab4cb02f1228ee216212
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-19 01:37:48 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
